### PR TITLE
Feat/reaction styling

### PR DESCRIPTION
**Pull request by OCannings**
Still a WIP - opening PR to create heroku preview app (https://juiceboxapp-staging-pr-73.herokuapp.com/)

### CODE REVIEW

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 1]
```diff
@@ -68,12 +190,38 @@
...
+.reaction-button {
+  background-color: #2f7791;
+  cursor: pointer;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  padding: 4%;
+
+  & > svg {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Unnecessary parent selector (&)

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#unnecessaryparentreference)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 2]
```diff
@@ -68,12 +190,38 @@
...
+    }
+
+    &.selected {
+      background-color: #999;
     }
   }
 }
+
+.reaction-button {
+  background-color: #2f7791;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, display, width, height, padding, cursor, background-color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 3]
```diff
@@ -68,12 +190,38 @@
...
-    margin: 0 10px;
+    width: 49%;
+    height: 0;
+    margin: 0;
     cursor: pointer;
+    padding-bottom: 50%;
+    background-repeat: no-repeat;
+    background-size: 85% 85%;
+    background-position: center center;
+    background-color: white;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `white` should be written in hexadecimal form as `#ffffff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 4]
```diff
@@ -68,12 +190,38 @@
 
   li {
     display: inline-block;
-    width: 100px;
-    margin: 0 10px;
+    width: 49%;
+    height: 0;
+    margin: 0;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered display, width, height, padding-bottom, margin, margin-bottom, cursor, background-color, background-repeat, background-position, background-size

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 5]
```diff
@@ -57,9 +176,12 @@
 .recorder-controls {
   display: flex;
 
-  button {
-    flex: 1;
-    margin: 0 10px;
+  .btn {
+    width: 100%;
+
+    & + .btn {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Unnecessary parent selector (&)

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#unnecessaryparentreference)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 6]
```diff
@@ -22,8 +134,15 @@
...
 
+.webcam-container {
+  background-color: white;
+  margin-bottom: 10px;
+}
+
 .webcam {
-  width: 200px;
+  width: 90%;
+  margin: 0 auto;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered width, padding, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 7]
```diff
@@ -22,8 +134,15 @@
   padding: 20px;
 }
 
+.webcam-container {
+  background-color: white;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `white` should be written in hexadecimal form as `#ffffff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered margin-bottom, background-color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 8]
```diff
@@ -1,11 +1,123 @@
...
+    content: ' ';
+    padding-bottom: 120%;
+    display: block;
+  }
+}
+
+.reaction-list {
+  .video-frame {
+    position: absolute;
+    width: 100%;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, display, width, height

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 9]
```diff
@@ -1,11 +1,123 @@
...
+  }
+
+  &:after {
+    content: ' ';
+    padding-bottom: 120%;
+    display: block;
+  }
+}
+
+.reaction-list {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Merge rule `.reaction-list` with rule on line 74

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#mergeableselector)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 10]
```diff
@@ -1,11 +1,123 @@
...
+    width: 22%;
+  }
+
+  @media (min-width: 1900px) {
+    width: 18%;
+  }
+
+  &:after {
+    content: ' ';
+    padding-bottom: 120%;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered content, display, padding-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 11]
```diff
@@ -1,11 +1,123 @@
...
+
+  @media (min-width: 1400px) {
+    width: 22%;
+  }
+
+  @media (min-width: 1900px) {
+    width: 18%;
+  }
+
+  &:after {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Begin pseudo elements with double colons: `::`

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 12]
```diff
@@ -1,11 +1,123 @@
...
+        content: ' ';
+        width: 100%;
+        height: 100%;
+        background-color: white;
+        opacity: 0.93;
+      }
+    }
+
+    .reactions-container {
+      display: flex;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, top, left, display, width, max-width, margin, transform

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 13]
```diff
@@ -1,11 +1,123 @@
...
+      height: 100%;
+
+      &:before {
+        position: fixed;
+        display: block;
+        content: ' ';
+        width: 100%;
+        height: 100%;
+        background-color: white;
+        opacity: 0.93;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: `0.93` should be written without a leading zero as `.93`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 14]
```diff
@@ -1,11 +1,123 @@
...
+      width: 100%;
+      height: 100%;
+
+      &:before {
+        position: fixed;
+        display: block;
+        content: ' ';
+        width: 100%;
+        height: 100%;
+        background-color: white;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `white` should be written in hexadecimal form as `#ffffff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 15]
```diff
@@ -1,11 +1,123 @@
...
+      display: block;
+      z-index: 10000;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+
+      &:before {
+        position: fixed;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered content, position, display, width, height, background-color, opacity

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 16]
```diff
@@ -1,11 +1,123 @@
...
+    .reactions-modal {
+      display: block;
+      z-index: 10000;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+
+      &:before {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Begin pseudo elements with double colons: `::`

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 17]
```diff
@@ -1,11 +1,123 @@
...
+}
+
 .reactions {
+  .reactions-modal {
+    display: none;
+  }
+
+  &.open {
+    .reactions-modal {
+      display: block;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, top, left, z-index, display, width, height

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 18]
```diff
@@ -1,11 +1,123 @@
...
+  background-color: #D32E6A;
+  padding: 0.5em;
+  cursor: pointer;
+}
+
+.close-btn {
+  position: absolute;
+  top: -6.4rem;
+  right: 0;
+  color: #D32E6A;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `#D32E6A` should be written as `#d32e6a`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexnotation)

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, top, right, font-size, color, cursor

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 25 possible new issues (including those that may have been commented here).
* 1 fixed issue! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/73.

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 19]
```diff
@@ -1,11 +1,123 @@
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  width: 100%;
+  background-color: #D32E6A;
+  padding: 0.5em;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: `0.5` should be written without a leading zero as `.5`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 20]
```diff
@@ -1,11 +1,123 @@
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  width: 100%;
+  background-color: #D32E6A;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `#D32E6A` should be written as `#d32e6a`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexnotation)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 21]
```diff
@@ -1,11 +1,123 @@
+.btn {
+  color: white;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `white` should be written in hexadecimal form as `#ffffff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered width, padding, color, text-align, text-transform, cursor, background-color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/reactions.js [hunk 22]
```diff
@@ -1,45 +1,99 @@
...
+
+  setOpen() {
+    this.setState({ open: true })
+  }
+
+  setClosed() {
+    this.setState({ open: false })
+  }
+
+  recorder = () => (
```
Comment (littleowllabs) [inline apps/juicebox_web/assets/js/reactions/reactions.js]: Parsing error: Unexpected token =

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 25 possible new issues (including those that may have been commented here).
* 1 fixed issue! 🎉

But beware that this branch is **[3 commits behind the `ItsJuice:master` branch](https://github.com/ItsJuice/juicebox/compare/ItsJuice:feat/reaction-styling...ItsJuice:master)**, and a review of an up to date branch would produce more accurate results.

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/73.

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 23]
```diff
@@ -39,41 +170,130 @@
...
+  & > svg {
+    object-fit: contain;
+    width: 100%;
+  }
+}
+
+.snap-icon {
+  width: 1.2em;
+  vertical-align: sub;
+  margin-right: 0.3em;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: `0.3` should be written without a leading zero as `.3`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 24]
```diff
@@ -39,41 +170,130 @@
...
+
+  & > svg {
+    object-fit: contain;
+    width: 100%;
+  }
+}
+
+.snap-icon {
+  width: 1.2em;
+  vertical-align: sub;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered width, margin-right, vertical-align

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 25]
```diff
@@ -39,41 +170,130 @@
...
+
+      &:nth-child(odd) {
+        margin-right: 2%;
+      }
+    }
+  }
+}
+
+.reaction-button {
+  background-color: #2f7791;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, display, width, height, padding, cursor, background-color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 26]
```diff
@@ -39,41 +170,130 @@
...
     }
   }
+
+  @media (min-width: $small) {
+    width: auto;
+
+    li {
+      width: 49%;
+      height: 0;
+      margin: 0;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered width, height, padding-bottom, margin, margin-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 27]
```diff
@@ -39,41 +170,130 @@
...
+      top: 50%;
+      left: 50%;
+      transition: all 0.4s;
+    }
+
+    &.selected {
+      background-color: #ccc;
+
+      .selected-indicator {
+        opacity: 1;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered top, left, width, opacity

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 28]
```diff
@@ -39,41 +170,130 @@
...
 
-    img {
-      width: 100%;
+    .selected-indicator {
+      position: absolute;
+      opacity: 0;
+      width: 0;
+      top: 50%;
+      left: 50%;
+      transition: all 0.4s;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: `0.4` should be written without a leading zero as `.4`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 29]
```diff
@@ -39,41 +170,130 @@
...
+    background-size: 85% 85%;
+    background-position: center center;
+    background-color: white;
+    position: relative;
 
-    img {
-      width: 100%;
+    .selected-indicator {
+      position: absolute;
+      opacity: 0;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, top, left, width, opacity, transition

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 30]
```diff
@@ -39,41 +170,130 @@
...
-    width: 100px;
-    margin: 0 10px;
     cursor: pointer;
+    width: $frame-width;
+    height: $frame-width;
+    margin-right: $frame-spacing;
+    background-repeat: no-repeat;
+    background-size: 85% 85%;
+    background-position: center center;
+    background-color: white;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `white` should be written in hexadecimal form as `#ffffff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 31]
```diff
@@ -39,41 +170,130 @@
...
+.frame-scroller {
+  overflow-x: auto;
+  overflow-y: hidden;
+  padding: 10px;
 
-  button {
-    flex: 1;
-    margin: 0 10px;
+  @media (min-width: $small) {
+    overflow: visible;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered padding, overflow

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 32]
```diff
@@ -39,41 +170,130 @@
...
+      & + .btn {
+        margin-right: 1em;
+        order: 0;
+      }
+    }
+  }
+}
+
+.frame-scroller {
+  overflow-x: auto;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered padding, overflow-x, overflow-y

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 33]
```diff
@@ -39,41 +170,130 @@
...
+    margin-bottom: 0.75em;
+  }
+
+  @media (min-width: $small) {
+    flex-direction: row;
+
+    .btn {
+      order: 1;
+
+      & + .btn {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Unnecessary parent selector (&)

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#unnecessaryparentreference)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 34]
```diff
@@ -39,41 +170,130 @@
...
   }
 }
 
 .recorder-controls {
   display: flex;
+  flex-direction: column;
+
+  .btn {
+    width: 100%;
+    margin-bottom: 0.75em;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: `0.75` should be written without a leading zero as `.75`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 35]
```diff
@@ -39,41 +170,130 @@
     content: ' ';
     display: block;
     visibility: hidden;
-    background-color: red;
+    background-color: #D32E6A;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `#D32E6A` should be written as `#d32e6a`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexnotation)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 36]
```diff
@@ -17,13 +146,15 @@
...
-  padding: 20px;
+.webcam-container {
+  background-color: white;
+  margin-bottom: 10px;
 }
 
 .webcam {
-  width: 200px;
+  width: 75%;
+  margin: 0 auto;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered width, padding, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 37]
```diff
@@ -17,13 +146,15 @@
   }
 }
 
-.reactions-container {
-  background-color: #f2f2f2;
-  padding: 20px;
+.webcam-container {
+  background-color: white;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `white` should be written in hexadecimal form as `#ffffff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)

Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered margin-bottom, background-color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 38]
```diff
@@ -1,11 +1,140 @@
...
+
+  @media (min-width: 1400px) {
+    width: 22%;
+  }
+
+  @media (min-width: 1900px) {
+    width: 17%;
+  }
+
+  &:after {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Begin pseudo elements with double colons: `::`

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 39]
```diff
@@ -1,11 +1,140 @@
...
+.reaction-list {
+  display: flex;
   width: 100%;
+  flex-flow: row wrap;
+}
+
+.reaction-list-item {
+  position: relative;
+  width: 100%;
+  margin-bottom: 4%;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, width, margin-right, margin-bottom

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 40]
```diff
@@ -1,11 +1,140 @@
...
+      height: 100%;
+
+      &:before {
+        position: fixed;
+        display: block;
+        content: ' ';
+        width: 100%;
+        height: 100%;
+        background-color: #f8f8f8;
+        opacity: 0.95;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: `0.95` should be written without a leading zero as `.95`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 41]
```diff
@@ -1,11 +1,140 @@
...
+  top: 20px;
+  right: 20px;
+  z-index: 200;
+  width: 31px;
+  height: 31px;
+  cursor: pointer;
+}
+
+.reactions-container {
+  padding: 20px;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Properties should be ordered position, top, left, display, width, max-width, padding, margin, transform

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 36 possible new issues (including those that may have been commented here).
* 3 fixed issues! 🎉

But beware that this branch is **[3 commits behind the `ItsJuice:master` branch](https://github.com/ItsJuice/juicebox/compare/ItsJuice:feat/reaction-styling...ItsJuice:master)**, and a review of an up to date branch would produce more accurate results.

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/73.

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 34 possible new issues (including those that may have been commented here).
* 3 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/73.

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 42]
```diff
@@ -39,41 +168,130 @@
...
-    width: 100px;
-    margin: 0 10px;
     cursor: pointer;
+    width: $frame-width;
+    height: $frame-width;
+    margin-right: $frame-spacing;
+    background-repeat: no-repeat;
+    background-size: 85% 85%;
+    background-position: center center;
+    background-color: #ffffff;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `#ffffff` should be written as `#fff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexlength)

File: apps/juicebox_web/assets/js/reactions/styles.scss [hunk 43]
```diff
@@ -17,13 +144,15 @@
   }
 }
 
-.reactions-container {
-  background-color: #f2f2f2;
-  padding: 20px;
+.webcam-container {
+  margin-bottom: 10px;
+  background-color: #ffffff;
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/styles.scss]: Color `#ffffff` should be written as `#fff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexlength)

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 6 possible new issues (including those that may have been commented here).
* 3 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/73.

**Comment by ljkbennett** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 3 possible new issues (including those that may have been commented here).
* 3 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/73.

**Review by ljkbennett — APPROVED** [review]

File: apps/juicebox_web/assets/js/reactions/reactions.js [hunk 44]
```diff
@@ -1,45 +1,100 @@
...
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/reactions/reactions.js:56]: Parsing error: Unexpected token =

**Pull request merged**